global-script.lua

```diff
@@ -3932,8 +3930,8 @@ end
 -- ensureCardInPlay moves the supplied card from a player's hand to a safe
 -- location, if it's in a hand.
 function ensureCardInPlay(card)
-    for color, _ in pairs(PlayerBags) do
-        for handIndex=1,numHandsPerPlayer do
+    for _, color in pairs(Player.getAvailableColors()) do
```

Comment:
why change where the colors are sourced from?

Reply:
I feel it makes the code more resilient (in fact, I think it would now work in any mod), since it will handle any players that have hands, not just the ones we have tables for. I don't expect it will make a practical difference, but since this isn't dealing with anything specific to this mod, it feels nicer to me to make it as generic as possible.

As an aside, I'd love to use `tts`'s ability to use [luabundle](https://github.com/Benjamin-Dobell/luabundle) (which is also (somewhat?) supported by the Atom plugin), to extract some of this generic code into their own files.